client/app/templates/Admin/Index/uploadOrgUsers.html.twig

```diff
@@ -14,30 +14,6 @@
 {% block pageContent %}
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-half hidden">
```

Comment:
do we actually want to get rid of this? I thought we were keeping the manual process as a backup if overnight job failed

Reply:
This is the OLD process, if you look below the other form is still present, which is what the current manual process uses.